examples/memnn/babi.py

```diff
@@ -0,0 +1,66 @@
+class Query(object):
+
+    def __init__(self, sentence, answer, fact):
+        self.sentence = sentence
+        self.answer = answer
+        self.fact = fact
+
+
+class Sentence(object):
```

Comment:
What is the motivation for wrapping `sentence` by this class (even though it only has one attribute and no other interface)?

Reply:
It is just for consistency. This dataset contains two types of lines, query and sentence. I need to use `Query` class (as you say namedtuple is OK) for the former. I need to use different data type for the latter.